src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int[] salaries = new int[names.length];
+        String[] stringDateFromArray = dateFrom.split("\\.");
+        String[] stringDateToArray = dateTo.split("\\.");
+        LocalDate dateStart = LocalDate.of(Integer.parseInt(stringDateFromArray[2]),
```

Comment:
you can parse date easier with DateFormatter, use it

Reply:
done